solana/gatekeeper-lib/src/service/GatekeeperService.ts

```diff
@@ -88,26 +87,26 @@ export class GatekeeperService {
    * @param {Transaction} transaction
    * @returns {Promise<string>}
    */
-  private async serializeBuiltTransaction(
+  private async signAndSerializeBuiltTransaction(
     transaction: Transaction
   ): Promise<string> {
     // recent blockhash and feepayer are required for serialization
-    // a real recent blockhash will be added in signAndSendTransaction
-    transaction.recentBlockhash = dummyBlockhash;
+    // a real recent blockhash will be added in sendSerializedTransaction
```

Comment:
Is this comment still accurate? It's adding the real blockhash now?

Reply:
removed comment